sklearn/datasets/samples_generator.py

```diff
@@ -611,22 +612,25 @@ def make_circles(n_samples=100, shuffle=True, noise=None, random_state=None,
         The integer labels (0 or 1) for class membership of each sample.
     """
 
-    if factor > 1 or factor < 0:
+    if factor >= 1 or factor < 0:
```

Comment:
Personally, I'm with you. But this latest source code change is not reviewed by anyone else. So need to confirm with core devs to see whether we need factor=1 in extreme case.

Reply:
Yes, definitely. If it was a typo in the docs then I'll revert the change and adjust the documentation. :)

Reply:
I this is fine.